src/Framework/Blocks/BlockModel.php

```diff
@@ -78,9 +78,7 @@ public function getAttributes(): array
      */
     public function setAttribute(string $name, $value): BlockModel
     {
-        if ($this->hasAttribute($name)) {
-            $this->attributes[$name] = $value;
-        }
+        $this->attributes[$name] = $value;
```

Comment:
Was this intentional?

Reply:
Yes! I notice that I had added that and now I was not being able to set new attributes programmatically what disallow us and any 3rd-party dev to add custom attributes to a block.

Reply:
Ahh good catch!